doc/manual.rst

```diff
@@ -251,6 +251,10 @@ Another advantage is that it frees the programmer from remembering
 the exact spelling of an identifier. The exception with respect to the first
 letter allows common code like ``var foo: Foo`` to be parsed unambiguously.
 
+Note that this rule also appies for keywords, meaning that ``notin`` is
```

Comment:
Typo, `appies for` -> `applies to`

Reply:
Fixed typo.

Btw, CIs were passing before, except on OSX where some unrelated tests were failing. (I'm merging it)